code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -96,9 +96,8 @@
 		if(reagents.total_volume > 30) // 30 equates to 3 SECONDS.
 			usr.visible_message(SPAN_NOTICE("[usr] prepares to gulp down [src]."), SPAN_NOTICE("You prepare to gulp down [src]."))
 		if(!do_after(usr, reagents.total_volume))
-			if(!Adjacent(usr))
-				return
-			standard_splash_mob(src, src)
+			standard_splash_mob(usr, usr)
```

Comment:
Would be nice if throwing around un-capped bottles would splash their contents as well, this is a nice change to enforce RP instead of walking and chugging
~~Unless you're a Hamon user~~